webapp/src/pages/Settings.tsx

```diff
@@ -548,8 +569,40 @@ const Settings: React.FC = () => {
     </FormGroup>
   );
 
+  const displayAnalysesCustomizationGeneralSection = () => (
+    <Box display="flex" justifyContent="flex-start" gap={5} alignItems="center">
+      <FormControl variant="standard" sx={{ m: 1, width: 120 }}>
+        <InputLabel id="supported-language-input-label">
+          supported_language
+        </InputLabel>
+        <Select
+          value={language ?? resultingConfig.language}
+          label="supported_language"
+          onChange={(event) =>
+            setLanguage(event.target.value as SupportedLanguage)
+          }
+        >
+          {SUPPORTED_LANGUAGES.map((language) => (
+            <MenuItem key={language} value={language}>
+              {language}
+            </MenuItem>
+          ))}
+        </Select>
+      </FormControl>
+      <Box display="flex" flexDirection="row" gap={1}>
+        <Warning color="warning" />
+        <Typography variant="subtitle2">
```

Comment:
That's not a subtitle.
```suggestion
        <Typography variant="body2">
```